addPlant.php

```diff
@@ -0,0 +1,137 @@
+<?php
+            require_once 'src/PlantModel.php';
+
+            if (isset($_POST['newPlantName'])&&
+                isset($_POST['newPlantScientificName'])&&
+                isset($_POST['newPlantFamily'])&&
+                isset($_POST['newPlantImage'])&&
+                isset($_POST['newPlantDescription'])){
+
+                $inputtedPlantName = $_POST['newPlantName']; 
+                $inputtedScientificName = $_POST['newPlantScientificName'];
+                $inputtedPlantFamily = $_POST['newPlantFamily'];
+                $inputtedImage = $_POST['newPlantImage'];
+                $inputtedDescription = $_POST['newPlantDescription'];
+                
+                if(strlen($inputtedPlantName)==0 ||
+                is_numeric($inputtedPlantName)) {
+                    echo "Please select use valid characters for the plant name.";
+                }
+                if(intval($inputtedPlantFamily)==0 ||
+                intval($inputtedPlantFamily < 4)) {
+                    echo "Please select a valid plant family.";
+                }
+                if(is_numeric($inputtedScientificName)) {
+                    echo "Please select use valid characters for the scientific name.";
+                }
+                if(strlen($inputtedImage)==0 || 
```

Comment:
You should always have a space around operators, so for example you should write `strlen($inputtedImage) == 0` instead of `strlen($inputtedImage)==0`